src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -118,6 +118,16 @@ public void ClusterOnDatabaseChanged(object sender, string dbName)
             // if deleted, unload / deleted and then notify leader that we removed it
         }
 
+        private void SendRachisCommandToLeader(DynamicJsonValue x)
```

Comment:
We even have the sync operation? I don't think we'll want to actually wait on that, would we?
And that lead to needing the thread pool.